### ISSUE TITLE
Resonance of the Faceted Crystal not finding other will types to split crytsals

### ISSUE THREAD

**Issue by MineTheGresh** [WayofTime/BloodMagic#1614]
#### Issue Description:
I am trying to create the 4 other aspects. Running  Resonance of the Faceted Crystal. Only getting raw crystals. 

BL Version is 1.12.2-2.3.3-101. Running in a mod pack but Blood Magic has not been tinkered with.

#### What happens:
I am running a Mob Farm
I am running Gathering of Forsaken Souls
I am running the Ritual of the Fractured Crystal.
The Demonic Crystaliser is placed on top of the master ritual stone.
The raw crystal is place on the Demonic Crystaliser.
The raw crystal grows chunks rapidly with this set up.

#### What you expected to happen:
Chunks of the different crystal aspects to grow.
Checked the Aura gauge and the chunk only has raw aura in it  is not generating other types of aura so I guess that is why I am not getting  other types of crystal chunk growing.

#### Steps to reproduce:

1. Build Mob farm.
2. Place Resonance of the Faceted Crystal ritual underneath.
3. Place Gathering of Forsaken Souls underneath that.
4. Place Ritual of the fractured crystal underneath that. 
5. Make sure Demon crucibles are burning raw crystal and filling aura into the chunk.
6. In hellfire forge create the crystal out of 4 raw crystal chunks.
7. Place Crystal on top of demonic crystaliser that is on to of the master ritual stone of the Resonance of the Faceted Crystal
8. Sit back and watch the crystal chunks grow. Unfortunately they are all 

____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: 
- Minecraft: 
- Forge:

**Comment by rudger-wolvram** [comment]
I'm not sure what you're trying to accomplish with WoS, it should be the Gathering of Forsaken Souls. The best way I found (note, my BM is 1 version behind yours if it's from the may 19th release) is to have it stacked (from bottom to top) Fractured, Gathering, Faceted. They all sit nicely into each other (may need to space MRS vertically to fit them as the MRS's won't touch). Cursed earth below or some other kind of spawner under the whole setup. 

When it's stacked the Gathering will hurt things below it, feeding crystal growth 5 above it. Fractured sitting below the gathering will break off pieces 7 above it (IIRC, I switched to Woot's rituals) which means the 3 block space above the Gathering ritual will be broken (you lose 2 levels from the gathering's reach). 

The Faceted will fit into the gathering such that the crystalizer on top of the Faceted MRS will not have the raw crystal there broken, allowing the growth and split to occur. Once that crystal has 4 spires on it, it should split off to the neighboring ritual stones as the 4 aspects (1 block down and cardinal out from the crystalizer crystal). 

From there you will need to let the raw crystal ontop of the crystalizer gow and split until you have enough aspects of will to feed back into the chunk(s) via 4 crucibles. They will then continue to grow from the raw and the will in the aura. Then when you have enough banked, you can make new aspect crystals and place them down in the growth area and as long as mobs are fed into it and there is will of the right types being fed into it, they will grow and break off. Obviously you'll need Zephyr or some other item collector to feed them back into your system. I used an absorption hopper from mob grinding, ender chest from ender storage, and item conduits from enderIO for least infrastructure/simplicity sake.

**Comment by MineTheGresh** [comment]
Thanks for the reply. 

Updated. My bad, it was the Gathering of Forsaken Souls. The raw crystals happily grow on the Crystaliser on top of the Faceted ritual all the way to seven spires. I am not seeing the splitting behaviour. The mob spawner is above faceted but in easy range of the Gathering ritual. Fractured sits underneath Gathering and will happily break crystals on Gathering. Fractured is not in range of Faceted does that matter? I have used a lever to switch off Fractured and I am manually splitting the 7 spires of the raw crystal on top of the Faceted ritual.

I did try and run Faceted in the chunk next to the GotFS ritual with a Demon Pylon to balance out aura but that had the same result, no splitting.
